release-team/role-handbooks/release-notes/editing-flow.md

```diff
@@ -0,0 +1,245 @@
+    - [krel release-notes --fix](#krel-release-notes--fix)
+      - [Starting the interactive mode](#starting-the-interactive-mode)
+      - [Reviewing and editing the release notes](#reviewing-and-editing-the-release-notes)
+      - [Exiting the review loop](#exiting-the-review-loop)
+    - [Submitting changes back](#submitting-changes-back)
+    - [Re-running the workflow](#re-running-the-workflow)
+      - [Modified Release Notes](#modified-release-notes)
+
+
+## Introduction
+
+Ensuring the production of a document comprised of well written and 
+informative release notes is one of the responsibilities of the 
+Release Team each cycle. 
+
+Release Engineering has developed various tools to help the Team 
+complete its duties. One of them is `krel` - the [Kubernetes Release
+Toolbox](https://github.com/kubernetes/release/tree/master/docs/krel).
+Krel has many subcommands that help the release process in many ways,
+among them the `release-notes` subcommand.
+
+This document details how to use krel to review and edit the 
+Kubernetes release notes during the cycle. 
+
+## Motivation
+
+The final Kubernetes Release Notes document is a large document composed of
+hundreds of entries that the contributors write into their Pull Requests. As
+an example, the [release notes document for Kubernetes 1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
+has 388 entries.
+
+Before v1.20, the Release Notes Team had to wait until code freeze to review
+and edit each Kubernetes release note in a short time. During the same
+timespan, the Team was expected to contact the sigs and edit the Major themes.
+Work was unbalanced with little to do most of the cycle and a packed 
+agenda during the last few weeks. 
+
+To balance the team's efforts evenly, tooling was developed to enable a new workflow.
+
+## Workflow Operation
+
+Reviewing the release notes involves running krel regularly to review new release
+notes in incoming pull requests, possibly editing some of them, and submitting
+changes back to `sig-release`.
+
+### `krel release-notes --fix`
+
+The main interactive editing flow is started with the --fix flag. The interactive
+mode will show the user all release notes that need to be reviewed. After
+reviewing each note, the user can edit the note's attributes using their editor of
+choice.
+
+The `--fix` flag is available when invoking krel with the `--create-draft-pr` option.
+This is an example invocation to edit the release notes for Kubernetes v1.19:
+
+```bash
+krel release-notes --create-draft-pr --org=MyGitHubOrg --fix --tag=v1.19.0
+```
+As with all `krel release-notes` subcommands, a GitHub token must be exported in
+an environment variable called `GITHUB_TOKEN`. The user should supply their GitHub
+organization where krel will push changes and from where it will create pull
+requests.
+
+After invoking the subcommand, krel will clone kubernetes/kubernetes and query GitHub
```

Comment:
Is it worth clarifying what "kubernetes/kubernetes" means? To someone new to the project it may not be obvious?

Reply:
OK, I've changed it to "will clone the `kubernetes/kubernetes` repository"